src/Umbraco.Web.UI.Client/src/views/components/blockcard/umb-block-card.html

```diff
@@ -1,10 +1,17 @@
-<div>
-    <div class="__showcase" ng-style="{'background-color':vm.blockConfigModel.backgroundColor, 'background-image': vm.styleBackgroundImage}">
-        <i ng-if="vm.blockConfigModel.thumbnail == null && vm.elementTypeModel.icon" class="__icon {{ vm.elementTypeModel.icon }}" ng-attr-style="{{'color:'+vm.blockConfigModel.iconColor+' !important'}}" aria-hidden="true"></i>
+
+    <div class="__showcase"
+        ng-class="{'--error':vm.elementTypeModel === null}"
+        ng-style="{'background-color': vm.blockConfigModel ? vm.blockConfigModel.backgroundColor : '', 'background-image': vm.styleBackgroundImage}">
+        <i ng-if="vm.blockConfigModel == null || vm.blockConfigModel.thumbnail == null" class="__icon {{ vm.elementTypeModel ? vm.elementTypeModel.icon : 'icon-block' }}" ng-attr-style="{{'color:'+vm.blockConfigModel.iconColor+' !important'}}" aria-hidden="true"></i>
```

Comment:
@nathanwoulfe we can probably use `<umb-icon>` here as well and I wonder if we still need `!important` to force this color style if using `<umb-icon>` component 👍